Rubberduck.Parsing/Symbols/DeclarationFinder.cs

```diff
@@ -777,22 +777,16 @@ private bool IsProjectGlobalDeclaration(Declaration candidateDeclaration, Declar
         {
             return candidateDeclaration.ProjectName == scopingDeclaration.ProjectName
                 && !(candidateDeclaration.ParentScopeDeclaration is ClassModuleDeclaration)
-                && (IsExplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration)
-                || IsImplicitPublicInOtherModule(candidateDeclaration, scopingDeclaration));
+                && (IsPublicInOtherModule(candidateDeclaration, scopingDeclaration));
         }
 
-        private bool IsExplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
+        private bool IsPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
         {
             return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
-                    && candidateDeclaration.Accessibility == Accessibility.Public;
-        }
-
-        private bool IsImplicitPublicInOtherModule(Declaration candidateDeclaration, Declaration scopingDeclaration)
-        {
-            return candidateDeclaration.ComponentName != scopingDeclaration.ComponentName
-                        && candidateDeclaration.Accessibility == Accessibility.Implicit
-                        && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)
-                        && !candidateDeclaration.IdentifierName.StartsWith("Option ");
+                        && (candidateDeclaration.Accessibility == Accessibility.Public
+                            || (candidateDeclaration.Accessibility == Accessibility.Implicit)
+                                && (candidateDeclaration.ParentScopeDeclaration is ProceduralModuleDeclaration)
```

Comment:
Fair enough then - let it be a *convenience function* exposed by the finder so that finder clients can more easily check for accessibility without taking a direct dependency on `AccessibilityCheck`.